server.js

```diff
@@ -7,10 +7,25 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const Thought = mongoose.model('Thought', {
+    text: {
+      type: String,
+      required: true,
+      minlength: 5,
+      maxlength: 140,
+    },
+    heart: {
+      type: Number,
+      default: () => 0
```

Comment:
I think here default can be just `default: 0` instead of using the `() => 0` syntax